bees/mumblebee/mumblebeefactory.go

```diff
@@ -0,0 +1,208 @@
+		},
+		{
+			Name:        "channel",
+			Description: "Channel to join on the Mumble server, blank for root",
+			Type:        "string",
+			Mandatory:   false,
+		},
+		{
+			Name:        "user",
+			Description: "Username to authenticate with Mumble server",
+			Type:        "string",
+			Mandatory:   true,
+		},
+		{
+			Name:        "password",
+			Description: "Password to use to connect to Mumble server",
+			Type:        "password",
+			Mandatory:   false,
+		},
+		{
+			Name:        "insecure",
+			Description: "Do not check server certificate",
+			Default:     true,
+			Type:        "bool",
+			Mandatory:   false,
+		},
+	}
+	return opts
+}
+
+// Events describes the available events provided by this Bee.
+func (factory *MumbleBeeFactory) Events() []bees.EventDescriptor {
+	events := []bees.EventDescriptor{
+		{
+			Namespace:   factory.Name(),
+			Name:        "message",
+			Description: "A message was received over Mumble",
+			Options: []bees.PlaceholderDescriptor{
+				{
+					Name:        "text",
+					Description: "The message that was received",
+					Type:        "string",
+				},
+				{
+					Name:        "user",
+					Description: "The user that sent the message",
+					Type:        "string",
+				},
+				{
+					Name:        "channels",
+					Description: "The channels where the message was received",
+					Type:        "string",
+				},
+			},
+		},
+	}
+	return events
+}
+
+// Actions describes the available actions provided by this Bee.
+func (factory *MumbleBeeFactory) Actions() []bees.ActionDescriptor {
+	actions := []bees.ActionDescriptor{
+		{
+			Namespace:   factory.Name(),
+			Name:        "move",
+			Description: "Moves to a new Channel",
+			Options: []bees.PlaceholderDescriptor{
+				{
+					Name:        "channel",
+					Description: "The channel to move in",
+					Type:        "string",
+					Mandatory:   true,
+				},
+			},
+		},
+		{
+			Namespace:   factory.Name(),
+			Name:        "play",
+			Description: "Play a sound file",
+			Options: []bees.PlaceholderDescriptor{
+				{
+					Name:        "filename",
+					Description: "The name of the file to play",
+					Type:        "string",
+					Mandatory:   true,
+				},
+			},
+		},
+		{
+			Namespace:   factory.Name(),
+			Name:        "send_to_user",
+			Description: "Sends a message to a user",
+			Options: []bees.PlaceholderDescriptor{
+				{
+					Name:        "user",
+					Description: "Which user to send the message to",
+					Type:        "string",
+					Mandatory:   true,
+				},
+				{
+					Name:        "text",
+					Description: "Content of the message",
+					Type:        "string",
+					Mandatory:   true,
+				},
+			},
+		},
+		{
+			Namespace:   factory.Name(),
+			Name:        "send_to_channel",
+			Description: "Sends a message to a channel",
+			Options: []bees.PlaceholderDescriptor{
+				{
+					Name:        "channel",
+					Description: "Which channel to send the message to",
+					Type:        "string",
+					Mandatory:   true,
```

Comment:
This should be false. If no channel gets provided, this bee will send to the current channel.